### PR TITLE
Add support for backup count in Vector Collection configuration [AI-145]

### PR DESCRIPTION
**Pull request by k-jamroz**
Backup count configuration for vector collection was added in https://github.com/hazelcast/hazelcast-client-protocol/pull/533

New parameters need #699 but the tests work with both 5.5 (skipped) and 6.0.

### CODE REVIEW

File: tests/integration/backward_compatible/proxy/vector_collection_test.py
Comment (k-jamroz) [reply tests/integration/backward_compatible/proxy/vector_collection_test.py:172]: yes, 5.5 server ignores backup counts in client message so it does not validate them

**Review by srknzl — APPROVED** [review]
I'm not familiar with vector collection stuff, so I cant approve for actual logic for this PR imo. so pls wait for Yuce approval

**Review by yuce — APPROVED** [review]

**Pull request merged**
